components/CreateCard.js

```diff
@@ -1,31 +1,67 @@
 import styled from "styled-components";
 import Image from "next/image";
 
+const CardStyle = styled.section`
+  min-width: 355px;
+  margin: 5%;
+  background-color: white;
+`;
+
+const PlantContainerStyle = styled.div`
+  border: 3px solid green;
+  border-radius: 0.7rem;
+  padding: 0.2rem 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+`;
+
+const PlaceContainerStyle = styled.div`
+  border: 3px solid darkred;
+  border-radius: 0.7rem;
+  padding: 0.2rem 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+`;
+
+const ContentContainerStyled = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 20px;
+`;
+
+const ListElementStyled = styled.p`
+  margin: 0;
+`;
+
 export default function CreateCard({ data, type }) {
   if (type === "plant") {
```

Comment:
You are using `type` to decide wether to render a place or a plant. These two objects do not have a lot in common, you only want to apply the same styling to the container. 
This makes your card component very unflexible - if you want to use it with a 3rd type of content you would have to add another `if` block and so on. 
A better approach  would be to create a container component (e.g. `Card`) that simply renders children. When you instantiate this component (e.g. in `placelist.js` or `plantlist.js`), you can then use the appropriate content component as child.